lldb/source/Plugins/Process/gdb-remote/GDBRemoteCommunicationServerLLGS.h

```diff
@@ -18,17 +18,13 @@
 #include "lldb/Host/common/NativeProcessProtocol.h"
 #include "lldb/Utility/RegisterValue.h"
 #include "lldb/lldb-private-forward.h"
+#include "LLDBServerPlugin.h"
 
 #include "GDBRemoteCommunicationServerCommon.h"
 
 class StringExtractorGDBRemote;
 
 namespace lldb_private {
-
-namespace lldb_server {
-  class LLDBServerPlugin;
-}
-
```

Comment:
Why is the #include needed here? If we don't use and instance of the `LLDBServerPlugin` in this file, then we shouldn't include the header file in here and we should stick with the forward declaration

Reply:
I'll try to move it to a cpp file

Reply:
@clayborg , I just pushed the fix